examples/Makefile.am

```diff
@@ -49,7 +48,6 @@ endif
 
 YAMI_DECODE_LIBS = \
 	$(YAMI_COMMON_LIBS) \
-	$(top_builddir)/decoder/libyami_decoder.la \
 	$(NULL)
 
 if BUILD_STATIC
```

Comment:
All of the whole-archive libraries specified by LDFLAGS inside the BUILD_STATIC conditionals should be changed to `$(top_builddir)/.libs/libyami.a` now.  I believe BUILD_STATIC is used in examples/Makefile.am and tests/Makefile.am.